resources/assets/less/landing.less

```diff
@@ -1,46 +1,80 @@
-@aboveMobile: ~"only screen and (min-width: 720px)";
-@aboveTablet: ~"only screen and (min-width: 1100px)";
+@import "./tailwind";
 
-@highlight-background-color: #648ec6;
-@secondary-background-color: #e4e7ec;
-
-@nav-margin:        50px; // Margin around nav area
-@content-margin:    20%;  // Horizontal margin around content
-@section-spacing:   50px; // Vertical padding around sections
-@font-size:         20px; // 1em font size
-
-@nav-links-spacing: 15px; // Horizontal spacing between nav links
-@intro-headroom:    50px; // Spacing between nav and intro content
-@intro-gutter:      50px; // Spacing between intro screenshot and claim
-@claim-spacing:     15px; // Spacing between claim items
-@features-spacing:  20px; // Vertical spacing between features
-@footer-spacing:    20px; // Padding around footer items
-
-* {
-    margin: 0;
-    padding: 0;
+html {
+    @apply .font-sans;
 }
-
 body {
-    font-family: 'Roboto', sans-serif;
-    font-size: @font-size;
-    margin: 0;
+    @apply .mx-auto;
+    max-width: 1440px;
+}
+h2 {
+    @apply .font-hairline;
+    font-size: 2rem;
+}
+.title-underline {
+    @apply .w-10 .border-solid .border-b-2 .border-blue-darkest .mx-auto .my-4;
+}
+.container {
+    max-width: 990px;
+    padding-left: 1.25rem;
+    padding-right: 1.25rem;
 }
 
-.cta-button {
-    background: black;
-    border-radius: 5px;
-    color: white;
+.btn {
+    @apply .border-0 .my-3 .text-base .py-2 .px-8 .rounded .no-underline .table;
+    line-height: 1.75rem;
+}
+.btn-cta {
+    @apply .bg-blue .text-white .text-lg;
+}
+.blue-overlay:before {
+    position: absolute;
+    content:" ";
```

Comment:
Another Tighten style rule is to always have a space to the right of the `:`

And another is to alphabetize the properties in any CSS block :)